docs/src/main/asciidoc/se/webserver/webserver.adoc

```diff
@@ -248,6 +248,7 @@ You can use *path pattern* instead of _path_ with the following syntax:
 * `/foo/{:\d+}` - Unnamed regular expression segment with a specified expression
 * `/foo/{+var}` - Convenience shortcut for `{var:.+}`
 * `/foo/{+}` - Convenience shortcut for unnamed segment with regular expression `{:.+}`
```

Comment:
Not directly related to this PR, but the "{+}" seems to be rendered as "{}". Should it be set to "{\+}"?